src/ngraph/axis_set.hpp

```diff
@@ -59,6 +59,11 @@ namespace ngraph
             static_cast<std::set<size_t>*>(this)->operator=(v);
             return *this;
         }
+
+        std::vector<int64_t> to_vector() const
```

Comment:
Why a vector of int64_t and not a vector of size_t? If there is a good reason then perhaps a comment? If not then return vector<size_t>

Reply:
In the long-run, we might want to switch AxisSet to int64_t instead of size_t to support negative axis specifications. Until that point, this is a hack to get int64_t from a size_t AxisSet.